README.md

```diff
@@ -1,8 +1,8 @@
 # Pull Request Labeler
 
 <p align="left">
-  <a href="https://github.com/actions/labeler/actions?query=workflow%3A%22Build+%26+Test%22++">
-    <img alt="build and test status" src="https://github.com/actions/labeler/actions/workflows/build_test.yml/badge.svg">
+  <a href="https://github.com/actions/labeler/actions/workflows/basic-validation.yml">
+    <img alt="basic validation status" src="https://github.com/actions/labeler/actions/workflows/basic-validation.yml/badge.svg">
   </a>
   <a href="https://david-dm.org/actions/labeler">
     <img alt="dependencies" src="https://status.david-dm.org/gh/actions/labeler.svg">
```

Comment:
This badge needs to be checked as well.

Reply:
Thank you for noticing this.
We can replace it with `https://img.shields.io/librariesio/github/actions/labeler`

Reply:
@MaksimZhukov Thank you for your comment! What link should I replace `href` with?